submitted_models/ctu_cras_norlab_spot_sensor_config_2/model.sdf

```diff
@@ -0,0 +1,2665 @@
+<?xml version='1.0' encoding='utf-8'?>
+<sdf version="1.6">
  <!-- CAUTION: This is an autogenerated file, DO NOT EDIT IT!
  It was generated by script update_robot_sdf and is based on urdf/spot.urdf.xacro source file.
  Do all modifications there and then run the update script to regenerate the SDF. -->
+  <model name="ctu_cras_norlab_spot_sensor_config_1">
+    <link name="base_link">
+      <inertial>
+        <mass>16</mass>
+        <inertia>
+          <ixx>0.0973333</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>1.02467</iyy>
+          <iyz>0</iyz>
+          <izz>1.04493</izz>
+        </inertia>
+      </inertial>
+      <collision name="base_link_collision">
+        <geometry>
+          <mesh>
+            <uri>../bosdyn_spot/meshes/body_collision.stl</uri>
```

Comment:
These relative paths are going to break when the models are uploaded to fuel, because of the additional version number directory.

I would propose uploading the current SPOT configuration, and adjusting these URIs to be correct for Fuel-based models.  After this PR is approved and lands, we can reupload the SPOT model with the final approved setup.

What do you think @peci1 ?

CC: @azeey, who is also reviewing a SPOT-derived model.

Reply:
I went ahead an uploaded as: https://app.ignitionrobotics.org/OpenRobotics/fuel/models/bosdyn_spot

Reply:
Thanks a lot, I got a bit overwhelmed by other tasks yesterday... I'll have a look at your other comments in a few hours

Reply:
Alright, I think this is the only thing remaining from my pov.

Reply:
I'll go ahead and do the URL unfurling also for references to CTU_CRAS_NORLAB_SPOT_SC1, which will be shortly invalid (until the model is uploaded to Fuel).

Reply:
I tested sensor config 1 with the new change and it loaded correctly.